tests/drivers/fprotect/positive/src/main.c

```diff
@@ -15,7 +15,7 @@
 #include <device.h>
 #include <drivers/flash.h>
 
-static uint8_t write_data[] = "Hello world";
+static uint8_t write_data[] = "HelloWorld";
```

Comment:
You added quotes when printing characters, especially to ensure correct handling with spacing.
So wouldn't it be nice to keep the space in `Hello World` so it's ensured it works as expected ?